timing.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./test.sh
```

Comment:
As suggested, now executing shell script rather than sourcing it. 

**[Note to self]**
To understand executing-vs-sourcing, see [this stackoverflow](https://superuser.com/questions/176783/what-is-the-difference-between-executing-a-bash-script-vs-sourcing-it)

----

@seanlaw 
we can try to just do unit testing part here but having coverage should be okay given the fact that the running time is small.

Reply:
I'm not sure. I was thinking that we DON'T do testing here and only do timing here and keep testing separate? 

Having said that, we can follow STUMPY and do both unit testing and coverage testing together inside of `test.sh`.

Reply:
Cool! Will separate them! `test.sh` 

Currently, `test.sh` follows STUMPY's design, and it does both unit testing and coverage.